code/modules/jobs/job_types/adventurer/types/combat/mage.dm

```diff
@@ -18,21 +18,22 @@
 	shoes = /obj/item/clothing/shoes/simpleshoes
 	armor = /obj/item/clothing/shirt/robe/colored/mage
 	belt = /obj/item/storage/belt/leather/rope
-	backr = /obj/item/storage/backpack/satchel
+	backl = /obj/item/weapon/polearm/woodstaff
+	backr = /obj/item/storage/backpack/backpack
 	beltr = /obj/item/storage/magebag/poor
 	beltl = /obj/item/reagent_containers/glass/bottle/manapot
 	backpack_contents = list(/obj/item/book/granter/spellbook/apprentice = 1, /obj/item/chalk = 1)
-	r_hand = /obj/item/weapon/polearm/woodstaff
 	if(H.mind)
 		H.adjust_skillrank(/datum/skill/misc/reading, 4, TRUE)
-		H.adjust_skillrank(/datum/skill/magic/arcane, 3, TRUE)
+		H.adjust_skillrank(/datum/skill/magic/arcane, 4, TRUE)
```

Comment:
keep in mind that with this, an old mage will start with Master Arcane, not a single adventurer has the possibility of having a combat skill as master with the exceptions being things like sneaking, pickpocketing

Reply:
That's a good catch, didn't realize that. Thank you